consensus/spos/bls/subroundStartRound.go

```diff
@@ -69,9 +72,11 @@ func checkNewSubroundStartRoundParams(
 	return err
 }
 
-// SetIndexer method set indexer
-func (sr *subroundStartRound) SetIndexer(indexer spos.ConsensusDataIndexer) {
-	sr.indexer = indexer
+// SetOutportHandler method set outport handler
+func (sr *subroundStartRound) SetOutportHandler(outportHandler outport.OutportHandler) {
+	sr.outportMutex.Lock()
+	sr.outportHandler = outportHandler
```

Comment:
please do:
1. change the function signature to: `SetOutportHandler(outportHandler outport.OutportHandler) error`
2. add a check as to not let setting a nil instance
3. remove the `check.IfNil(sr.outportHandler)` check on L209

These should be done in order to slightly increase the performance and to align with the rest of the project (wherewe do not continuously check for nils)